pkg/controllers/binding/common.go

```diff
@@ -154,18 +154,18 @@ func mergeTargetClusters(targetClusters []workv1alpha2.TargetCluster, requiredBy
 
 func mergeLabel(workload *unstructured.Unstructured, binding metav1.Object, scope apiextensionsv1.ResourceScope) map[string]string {
 	var workLabel = make(map[string]string)
-	util.MergeLabel(workload, util.ManagedByKarmadaLabel, util.ManagedByKarmadaLabelValue)
+	util.ReplaceLabel(workload, util.ManagedByKarmadaLabel, util.ManagedByKarmadaLabelValue)
```

Comment:
fells not right, as the workload might not have the label, how does `replace`?

Reply:
Any suggestion?

Reply:
I'm thinking that MergeLabel is the  appropriate name. The original MergeAnnotation function may not be necessary:

https://github.com/karmada-io/karmada/blob/4e2548250b19a8f8148c004defa02b0e99f5a3e7/pkg/util/annotation.go#L29-L40